src/utils.rs

```diff
@@ -23,7 +23,7 @@ use crate::{
     bot::State,
     db,
     request_handlers::{shop::ShopResponse, AppInfo, WexbdcManifest},
-    REVIEW_HELPER_XDC, SHOP_XDC, SUBMIT_HELPER_XDC,
+    REVIEW_HELPER_XDC, STORE_XDC, SUBMIT_HELPER_XDC,
 };
```

Comment:
A loose agreement with @hpk42 is, that we keep the current naming as long as possible and only before actually releasing the first MVP where we then make a final renaming PR. (@hpk42 and @r10s already discussed totally different naming so we should not preemptively  change things)
This is fine for now though.